heft-plugins/heft-typescript-plugin/src/TypeScriptBuilder.ts

```diff
@@ -534,15 +499,17 @@ export class TypeScriptBuilder {
     //#endregion
 
     //#region EMIT
-    const filesToWrite: IFileToWrite[] = [];
-
-    const writeFileCallback: TTypescript.WriteFileCallback = (filePath: string, data: string) => {
-      filesToWrite.push({ filePath, data });
-    };
-
     const { changedFiles } = configureProgramForMultiEmit(innerProgram, ts, this._moduleKindsToEmit, mode);
 
-    const emitResult: TTypescript.EmitResult = genericProgram.emit(undefined, writeFileCallback);
+    const emitResult: TTypescript.EmitResult = genericProgram.emit(
```

Comment:
Maybe add a comment for what's going on here? All the `undefined`s seem suspicious

Reply:
They're just there to match the arity of the underlying function.